toolchain/args/BUILD.bazel

```diff
@@ -64,7 +53,7 @@ cc_args(
         "@rules_cc//cc/toolchains/actions:cpp_link_executable",
     ],
     args = [
-        "-static",
+        "-static-pie",
```

Comment:
I fixed musl in the same time since we were not producing pie executable for musl)